From c39c2357e360dd7e70d89885483677783a5ef20a Mon Sep 17 00:00:00 2001
From: Neha Bisht <quic_nbisht@quicinc.com>
Date: Tue, 24 Jan 2023 13:21:24 +0530
Subject: [PATCH 253/500] skb_recycler: Enable reycler debug api under
 appropriate macro

Enable skb fast recycler debug api under reycler and debug check
macros

Change-Id: I0386ef6391761d2804f06b29b1d886b8ed0d117a
Signed-off-by: Neha Bisht <quic_nbisht@quicinc.com>
---
 include/linux/skbuff.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/linux/skbuff.h b/include/linux/skbuff.h
index 27ebc109751a..fce97f3d3045 100644
--- a/include/linux/skbuff.h
+++ b/include/linux/skbuff.h
@@ -1392,7 +1392,11 @@ static inline int skb_pad(struct sk_buff *skb, int pad)
 }
 #define dev_kfree_skb(a)	consume_skb(a)
 #define dev_kfree_skb_list_fast(a)	consume_skb_list_fast(a)
+#if defined(SKB_FAST_RECYCLABLE_DEBUG_ENABLE) && defined(CONFIG_SKB_RECYCLER)
 #define dev_check_skb_fast_recyclable(a)       check_skb_fast_recyclable(a)
+#else
+#define dev_check_skb_fast_recyclable(a)
+#endif
 
 int skb_append_pagefrags(struct sk_buff *skb, struct page *page,
 			 int offset, size_t size, size_t max_frags);
-- 
2.34.1

